mri/operators/linear/wavelet.py

```diff
@@ -13,6 +13,7 @@
 
 
 # Package import
+from .._base import OperatorBase
```

Comment:
Watch out for the name change in `base`.

Reply:
yup